solutions/instance/README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud-Object-Storage for IBM Cloud - Financial Services Cloud solution
+
+This architecture creates an Cloud-Object-Storage and bucket on IBM Cloud® and supports provisioning the following resources:
```

Comment:
Instance DA does not create bucket, so recheck and modify the content accordingly.